README.md

```diff
@@ -1,59 +1,6 @@
 # Ansible KubeVirt Modules
 
-[Ansible](https://github.com/ansible/ansible) modules for [KubeVirt](https://github.com/kubevirt/kubevirt) management.
-
-## Contents
-
-- `lib`: Ansible modules files for KubeVirt management
-    - `kubevirt_vm`: Manage virtual machines
-    - `kubevirt_rs`: Manage virtual machines replica sets
-- `tests`: Ansible playbook examples and unit tests
-
-## Requirements
-
-- Ansible >= 2.7
-- [OpenShift Python client](https://github.com/openshift/openshift-restclient-python)
-
-## Installation and usage
-
-1. Install the modules:
-    1. From GitHub:
-
-        ```shell
-        $ git clone https://github.com/kubevirt/ansible-kubevirt-modules
-        ```
-
-    2. From [Ansible Galaxy](https://galaxy.ansible.com/kubevirt/kubevirt-modules/)
-
-        ```shell
-        $ ansible-galaxy install -p <roles_path> kubevirt.kubevirt-modules
-        ```
-
-2. Setting up the environment
-
-```shell
-$ export ANSIBLE_MODULE_UTILS=<module_path>/lib/ansible/module_utils
-$ export ANSIBLE_LIBRARY=<module_path>/lib/ansible/modules
-```
-
-> **NOTE:** These settings can instead be added to *ansible.cfg* as done in [test/ansible.cfg](tests/ansible.cfg)
-
-3. A working [Kubernetes configuration](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) is also required. It can also be created by issuing `oc login` if using KubeVirt with OpenShift.
-
-4. [Install OpenShift Python SDK](https://github.com/openshift/openshift-restclient-python#installation)
-
-5. Once installed, add it to a playbook:
-
-```yaml
----
-- hosts: localhost
-  roles:
-    - role: ansible-kubevirt-modules
-      install_python_requirements: no
-    - role: hello-underworld
-```
-
-Because the role is referenced, the `hello-underworld` role is able to make use of the kubevirt modules.
+Modules have been moved to Ansible repository. This repo now only hold the integration tests of the modules.
```

Comment:
Would be good to have a link here to the modules.

Reply:
added